src/routes/RequestsRoute.js

```diff
@@ -1365,6 +1519,7 @@ class RequestsRoute extends React.Component {
               renderFilters={this.renderFilters}
               resultIsSelected={this.resultIsSelected}
               onFilterChange={this.handleFilterChange}
+              sortableColumns={['requestDate', 'title', 'year', 'itemBarcode', 'type', 'requestStatus', 'position', 'requester', 'requesterBarcode', 'proxy']}
```

Comment:
Should we add here 'Effective call number string' and 'Pickup service point' fields? Or it should not be included to this array?